docs/topics/api/signing.rst

```diff
@@ -14,15 +14,33 @@ as multi-part formdata. This will create a pending version on the
 add-on and will prevent future submissions to this version unless
 validation or review fails.
 
-Example::
+If the upload succeeded then it will be submitted for
+validation and you will be able to check its status.
 
-    curl https://addons.mozilla.org/en-US/firefox/api/v3/addons/my-addon/versions/1.0/
-         -XPUT --form 'upload=@build/my-addon.xpi' -H 'Authorization: JWT <jwt-token>'
+.. http:put:: /en-US/firefox/api/v3/addons/(add-on-guid)/versions/(version)/
 
-The response will be the same as the :ref:`check-status` response.
+    **Request:**
 
-If your upload has the right metadata then it will be submitted for
-validation and you will be able to check its status.
+    .. sourcecode:: bash
+
+        curl https://addons.mozilla.org/en-US/firefox/api/v3/addons/my-addon/versions/1.0/
+            -XPUT --form 'upload=@build/my-addon.xpi' -H 'Authorization: JWT <jwt-token>'
+
+    :param addon-guid: the GUID for the add-on.
+    :param version: the version of the add-on.
+    :form upload: the add-on being uploaded.
+    :reqheader Content-Type: multipart/form-data
+
+    **Response:**
+
+    The response body will be the same as the :ref:`check-status` response.
+
+    :statuscode 201: new add-on and version created.
+    :statuscode 202: new version created.
+    :statuscode 400: an error occurred, check the `error` value in the JSON.
+    :statuscode 403: authentication failed, or you do not own this add-on.
```

Comment:
You should be able to get both. 401 should happen if your jwt is invalid or missing while 403 will happen if you're authenticated but don't have permissions to this add-on.